README.md

```diff
@@ -190,3 +231,24 @@ $ curl http://localhost:8000/0/tx-history
   }
 }
 ```
+
+### Stop a wallet
+
+Stop the wallet and its connections and remove it from the store.
```

Comment:
What's the store?

Reply:
It's the wallets object the have all the wallet instances. Maybe it's not clear

Reply:
Yeah, it's not mentioned anywhere else. Also, it seems like an implementation detail. Not sure if it should be mentioned.

Reply:
I think it is the "set of wallets available". Maybe we should say "and remove it from the available wallets.".